shared/redux/rootReducers.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux'
+
+import mapReducer from './mapReducer'
+
+const rootReducers = combineReducers({
+  mapReducer
```

Comment:
Can we change this to a more appropriate name? We don't need to suffix `reducer` at the end.